operations/validate.go

```diff
@@ -174,7 +175,7 @@ func loadProjectIntoWithValidation(ctx context.Context, data []byte, opts *model
 	if err != nil {
 		errs = append(errs, validator.ValidationError{
 			Level:   validator.Error,
-			Message: err.Error(),
+			Message: fmt.Sprintf(err.Error(), "creating project config"),
```

Comment:
Seems like this has to be `errors.Wrap(err, "creating project config").Error()` or `fmt.Sprintf("creating project config: %s", err)` since `err.Error()` is not a formatting string.

Reply:
I removed this because i didn't mean for this to be part of this PR